src/components/Logo/style.module.scss

```diff
@@ -0,0 +1,6 @@
+@use '../../themes/theme.scss' as *;
+
+.Logo {
```

Comment:
You are not using this class anywhere and also it shouldn't start with upper case